nautobot/docs/administration/nautobot-server.md

```diff
@@ -291,22 +291,22 @@ This will run the following management commands with default settings, in order:
 !!! note
     Commands listed here that are not covered in this document here are Django built-in commands.
 
-`--no-clearsessions`  
```

Comment:
These spaces were purposefully here. Without the double-spaces, the text below will appear on the same line.

Reply:
![image](https://user-images.githubusercontent.com/17092348/185924475-61550e51-4e95-4046-9705-b6a712e2316a.png)

Reply:
I don't like the idea of putting  literal `<br/>` seeing as long as we maintain proper whitespace it should be fine.

Reply:
I too don't like the literal `<br/>` and I believe that is against markdownlint.

Reply:
Can we use 2 line breaks instead of trailing whitespace or `<br>`